std/os/netbsd/errno.zig

```diff
@@ -0,0 +1,134 @@
+pub const EPERM = 1; // Operation not permitted
+pub const ENOENT = 2; // No such file or directory
+pub const ESRCH = 3; // No such process
+pub const EINTR = 4; // Interrupted system call
+pub const EIO = 5; // Input/output error
+pub const ENXIO = 6; // Device not configured
+pub const E2BIG = 7; // Argument list too long
+pub const ENOEXEC = 8; // Exec format error
+pub const EBADF = 9; // Bad file descriptor
+pub const ECHILD = 10; // No child processes
+pub const EDEADLK = 11; // Resource deadlock avoided
+// 11 was EAGAIN
```

Comment:
What do you mean "was"? It seems to be 35 now.
When did it change? Why did it change?

Reply:
It is a comment from the original file in /usr/include/sys/errno.h, same thing on OpenBSD.